CONTRIBUTING.md

```diff
@@ -100,18 +101,26 @@ Test samples are kept in `/test/xxx/samples` folder.
 > PREREQUISITE: Install chromium via playwright by running `pnpm playwright install chromium`
 
 1. To run test, run `pnpm test`.
-1. To run test for a specific feature, you can use the `-g` (aka `--grep`) option. For example, to only run test involving transitions, run `pnpm test -- -g transition`.
+1. To run a particular test suite, use `pnpm test <suite-name>`, for example:
+
+   ```bash
+   pnpm test validator
+   ```
 
-##### Running solo test
+1. To run an individual test _within_ a test suite, use `FILTER=<test-name> pnpm test <suite-name>`, for example:
```

Comment:
Could we also document the vitest way of doing it? I find that easier to remember as it works on all my other projects as well (including SvelteKit)